src/systems/apply_link_wrench/ApplyLinkWrench.cc

```diff
@@ -163,7 +162,7 @@ void ApplyLinkWrench::Configure(const Entity &_entity,
 
     msg.mutable_entity()->set_name(elem->Get<std::string>("entity_name"));
 
-    auto typeStr = elem->GetElement("entity_type")->Get<std::string>();
+    auto typeStr = elem->FindElement("entity_type")->Get<std::string>();
```

Comment:
Check the element returned since `FindElement` may return `nullptr`?

Reply:
It's checked with `HasElement` above so it should be fine.